mscr-ui/src/pages/index.tsx

```diff
@@ -33,8 +33,18 @@ interface IndexPageProps extends CommonContextState {
 
 export default function IndexPage(props: IndexPageProps) {
   const { t } = useTranslation('common');
+  const theme = createTheme({
```

Comment:
We already use ThemeProvider in the Layout, providing the lightTheme from yti-common-ui/theme. These typographies look the same as the default theme has https://vrk-kpa.github.io/suomifi-ui-components/#/Foundations/Theme/Default%20Theme. Why do these need to be defined here as if they're a custom theme? Was something changed with updating the suomifi-ui-components?

Reply:
This might be redundant here. This is needed though in components using MUI components. Otherwise for it's using MUI default font.